code/datums/datacore.dm

```diff
@@ -492,6 +492,7 @@ GLOBAL_VAR_INIT(record_id_num, 1001)
 		if("Syndicate Nuclear Operative")
 			clothes_s = new /icon('icons/mob/clothing/under/syndicate.dmi', "syndicate_s")
 			clothes_s.Blend(new /icon('icons/mob/clothing/feet.dmi', "jackboots"), ICON_UNDERLAY)
+<<<<<<< HEAD
```

Comment:
Oh you utter dingus

Reply:
THAT'S THE MERGE THE MASTER COMMIT THAT I ACCIENDTLY PUSHED WITH DECOFLICTING!